cypress/integration/percy/article.web.spec.js

```diff
@@ -14,9 +14,8 @@ describe('For WEB', function() {
         it(`It should load ${designType} articles under the ${pillar} pillar`, function() {
             // Prevent the Privacy consent banner from obscuring snapshots
             cy.setCookie('GU_TK', 'true');
-            // Fix the location to UK (for edition)
-            cy.setCookie('GU_EDITION', 'UK');
-            cy.visit(`Article?url=${url}`, fetchPolyfill);
+            // Make the request, forcing the location to UK (for edition)
+            cy.visit(`Article?url=${url}?_edition=UK`, fetchPolyfill);
```

Comment:
Huh, I'm surprised the second `?` here works. I assume that's supposed to be a query string arg of `url`? In which case I'd expect it to need to be URL encoded. But if it works then nevermind! 🤷‍♂

Reply:
On one level is should be encoded, if it were then we'd be able to pass a complete url string in, including multiple query params. But encoding it in the browser when we do manual testing would be a chore so I left in this, somewhat invalid, state.